downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -58,6 +58,17 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
 
+#### `data.jsondata` packages
```

Comment:
Have to go under Ballerina library updates? Currently under Developer tools updates.